cli/jobs/pipelines/automl/cli-automl-classification-task-bankmarketing-pipeline/components/src/preprocess.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+import datetime
+from pathlib import Path
+import yaml
+from mltable import load
+
+
+def parse_args():
+    # setup arg parser
+    parser = argparse.ArgumentParser()
+
+    # add arguments
+    parser.add_argument("--train_data", type=str)
+    parser.add_argument("--validation_data", type=str)
+    parser.add_argument("--preprocess_train_data", type=str)
+    parser.add_argument("--preprocess_validation_data", type=str)
+    # parse args
+    args = parser.parse_args()
+    print("args received ", args)
+    # return args
+    return args
+
+
+def get_preprocessed_data(dataframe):
+    """
+    Do preprocessing as needed
+    Currently we are just passing pandas dataframe as it is
+    """
+    return dataframe
+
+
+def main(args):
+    """
+    Preprocessing of training/validation data
+    """
+    train_data_table = load(args.train_data)
+    train_dataframe = train_data_table.to_pandas_dataframe()
+    preprocessed_train_dataframe = get_preprocessed_data(train_dataframe)
+
+    # write preprocessed train data in output path
+    preprocessed_train_dataframe.to_csv(
+        args.preprocess_train_data + "/bank_marketing_train_data.csv",
```

Comment:
>args.preprocess_train_data + "/bank_marketing_train_data.csv", [](http://example.com/codeflow?start=8&length=62)

Please use os.path.join() instead for all path joins.

Reply:
Done !! Everywhere